lxd/patches.go

```diff
@@ -2504,6 +2505,187 @@ func patchStorageApiCephSizeRemove(name string, d *Daemon) error {
 	return nil
 }
 
+func patchDevicesNewNamingScheme(name string, d *Daemon) error {
+	cRegular, err := d.db.ContainersList(db.CTypeRegular)
+	if err != nil {
+		logger.Errorf("Failed to retrieve containers from database")
+		return err
+	}
+
+	cSnapshots, err := d.db.ContainersList(db.CTypeSnapshot)
+	if err != nil {
+		logger.Errorf("Failed to retrieve snapshots from database")
+		return err
+	}
```

Comment:
Snapshots can't have devices.

Reply:
What about the snapshots local config? Are the device entries removed? How do you restore a container including it's devices from a snapshot then?

Reply:
removed